docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Tutorial to distribute an OCI artifact to Zot Registry using ORAS CLI
+sidebar_position: 0
+---
+
+# Tutorial to Install Zot Registry and Distribute OCI Artifacts to Zot Registry using ORAS CLI
+
+Before we tackle how to distribute an OCI artifact to an OCI registry, let us first take a look at what we mean by distributing OCI artifacts, why we need it and what OCI registries are. 
+
+OCI registries are like a single location where container images and other artifacts are stored and made available to everyone. Distributing OCI artifacts refers to when artifacts are pushed to these registries. These artifacts can then be pulled (or retrieved) by someone else and used in their environments. These images often help with packaging and distributing software applications.
+
+We will be using zot registry in this guide. Zot registry is an OCI-native container registry for distributing container images and OCI artifacts.
```

Comment:
I would also call out that this setup is going to be plain http and only for testing purposes as this is insecure. For TLS setup follow - https://github.com/project-zot/zot/blob/main/examples/README.md#network

Reply:
Thank you, I have added it to a note at the end of the tutorial. Is adding the link enough or should I cover more in the tutorial?